### DISCUSSION TITLE
Compilar ejecutable

### DISCUSSION THREAD

**Discussion by jmlahoz** [agarsev/signario-guillermo#12] [General]
@agarsev 
Entiendo que los cambios de los issues cerrados ya están arreglados en el código pero no agregados a un nuevo ejecutable con esas modificaciones incorporadas. ¿Hay alguna manera de que yo descargue desde github el ejecutable que corresponde al código tal cual está en un momento? ¿O tengo que esperar a que me mandes una nueva versión compilada?

**Comment by agarsev** [comment]
(Movido a "discusiones")

**Comment by agarsev** [comment]
Efectivamente, hasta que yo compilo y subo la versión los cambios no están disponibles.

Estaba esperando a hacer un paquete más completo, pero ya que preguntas voy a subir la versión v0.3.1 con todas las issues del signotador, y sigo mientras tanto con el merge de bases de datos.

**Comment by agarsev** [comment]
#13

**Comment by jmlahoz** [comment]
@agarsev 
Genial, ¿me avisas cuando esté?
¡O sea que era aquí! Ya tenía yo la intuición de que esto no era un "issue" en el mismo sentido que los otros, pero no sabía dónde ponerlo, jejeje. Me gusta ir aprendiendo a usar el github "como se debe".

**Comment by agarsev** [comment]
Bueno, es bastante nuevo esto de las discusiones, hasta ahora se hacía en issues. Yo tampoco tengo muy claro dónde se responde, si a cada comentario o abajo del todo jaj.

La versión 0.3.1 con los cambios que pediste del signotador ya está, te aviso cuando salga la 0.4 con la mezcla de BBDDs.

Por cierto, no sé tampoco cuándo hay que @-mencionarse o no, te llega notificación de este mensaje?
